### PR TITLE
Rework the frontend CSS, JS, and HTML

### PR DESCRIPTION
**Pull request by felipesere**
This PR touches all aspects of the fronend:
* simplify the CSS, moving away from SASS and specific components
* simplify the HTML, using data-testids for the testing and not real classes
* Rearrange the JS to work with a single Modal

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -0,0 +1,579 @@
...
+  /* space 2 */
+  --s2: 10px;
+
+  --background: #f6f7f8;
+  --darker-background: #dadee2;
+  --darkest-background: #8694a1;
+
+  --text-color: #495561;
+
+  --primary: #27a8e0;
```
Comment (felipesere) [inline assets/css/main.css]: @fabientownsend recommends to decouple the names of the CSS vars from the actual colors names, but more for their intent, as described here: https://material.io/design/color/the-color-system.html#color-theme-creation

File: assets/css/main.css [hunk 2]
```diff
@@ -0,0 +1,579 @@
...
+  overflow: hidden;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 20;
+}
+
+.modal.is-active {
```
Comment (fabientownsend) [inline assets/css/main.css]: I'm not quire sure what approach take https://every-layout.dev/rudiments/modular-scale/ when it came to state, but you could consider http://getbem.com/ where you would have something like
``` css
.modal--active {}
```
it avoid you to chain class name and easily see the style that belongs together.

File: assets/css/main.css [hunk 3]
```diff
@@ -0,0 +1,579 @@
...
+  transform: translate(-50%, -50%);
+  background-color: #f5f5f5;
+}
+
+.modal-card .row {
+  justify-content: space-between;
+  margin: var(--s2);
+}
+
+.modal-body {
```
Comment (fabientownsend) [inline assets/css/main.css]: I find quite hard to understand what could be the difference between `.modal-body` `.modal-background`

File: assets/js/dashboardModal.js [hunk 4]
```diff
@@ -1,33 +1,53 @@
...
+        let element = findNodeWith(e.target, "data-questionnaire-id")
+
+        const questionnaireId = element.getAttribute("data-questionnaire-id");
+        const mentee = element.getAttribute("data-mentee-name");
+
+        $('#modal-title').text = `Confirm for ${mentee}`
+
+        toggle()
+
+        $(".remove").on("click", () => {
```
Comment (fabientownsend) [inline assets/js/dashboardModal.js]: Are you using JQuery only for the selector?

File: assets/js/fquery.js [hunk 5]
```diff
@@ -1,10 +1,8 @@
-export const all = (selector, event, listener) => {
-  document.querySelectorAll(selector).forEach(e => {
-    e.addEventListener(event, listener);
-  });
+export const forEach = (selector, event, listener) => {
```
Comment (fabientownsend) [inline assets/js/fquery.js]: You didn't have any issue to debug as anonymous function doesn't display any name in the stack trace?

File: assets/js/dashboardModal.js (hunk 4 above)
Comment (felipesere) [reply assets/js/dashboardModal.js]: Its fQuery!

File: assets/css/main.css (hunk 3 above)
Comment (felipesere) [reply assets/css/main.css]: Renamed it to darkened-modal-backdrop

**Pull request merged**